react-with-graphql-tutorial.md

```diff
@@ -1492,7 +1495,7 @@ class App extends Component {
 }
 {{< /highlight >}}
 
-The argument is simply passed to the `getIssuesOfRepository()` function which makes the GraphQL API request and returns the promise with the query result. Now check the other functions that call the `onFetchFromGitHub()` class method. They don't make use of the second argument and thus the cursor parameter will be `undefined` when it's passed to the GraphQL API call. That's fine, because as you will see next, either the query uses the cursor as argument to fetch the next page of a list or it fetches the initial page of a list by having the cursor not defined at all.
+The argument is simply passed to the `getIssuesOfRepository()` function, which makes the GraphQL API request and return the promise with the query result. Check the other functions that call the `onFetchFromGitHub()` class method, and notice hwo they don't make use of the second argument, so the cursor parameter will be `undefined` when it's passed to the GraphQL API call. Either the query uses the cursor as argument to fetch the next page of a list, or it fetches the initial page of a list by having the cursor not defined at all:
```

Comment:
"which makes the GraphQL API request and return the promise with the query result" to "which makes the GraphQL API request and returns the promise with the query result"

Reply:
"and notice hwo they don't make use of the second" to "and notice how they don't make use of the second"